.github/workflows/maintenance-label-wip.yml

```
@@ -0,0 +1,89 @@
+#   - removed once no changes are requested AND every conversation is resolved.
+#
+# The label is recomputed from the PR's live state (reviewDecision + review
+# threads) on every review / thread event, so the result is the same no matter
+# which event fired and running it twice is harmless (idempotent).
+
+run-name: 'WIP label - PR #${{ github.event.pull_request.number }}'
+
+on:
+  pull_request_review:
+    types: [submitted, dismissed]
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)